app/importers/mahonia_record_importer.rb

```diff
@@ -8,6 +8,16 @@ class MahoniaRecordImporter < Darlingtonia::RecordImporter
 
     ##
```

Comment:
We need representative data containing embargoes in order to know how to find the values Hyrax requires.

Reply:
:+1: